src/sentry/static/sentry/app/utils/queryString.tsx

```diff
@@ -33,12 +33,12 @@ type QueryValue = string | string[] | undefined | null;
 export function appendTagCondition(
   query: QueryValue,
   key: string,
-  value: string
+  value: null | string
```

Comment:
The old type was lies. In reality we got `null` here sometimes.

Reply:
Nice